src/components/Card.js

```diff
@@ -1,9 +1,19 @@
 export default class Card {
-  constructor({ data, cardSelector, handleImageClick }) {
+  constructor({
+    data,
+    cardSelector,
+    handleImageClick,
+    handleLikeToggle,
+    handleDeleteConfirm,
+  }) {
     this._name = data.name;
     this._link = data.link;
+    this._id = data._id; // Assuming 'id' is part of the data object
+    this._isLiked = data.isLiked || false; // Default to false if not provided
+    this._handleLikeToggle = handleLikeToggle; // Function to handle like toggle
     this._cardSelector = cardSelector;
     this._handleImageClick = handleImageClick;
+    this._handleDeleteConfirm = handleDeleteConfirm;
```

Comment:
* When I reload the page, all **black** hearts become **transparent** again.

Please, add ` this._updateLikeState();` in line 35 to toggle the color